src/style.css

```diff
@@ -0,0 +1,29 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  list-style: none;
+  padding: 0;
+  justify-content: center;
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 :nth-child(-n + 1),
+.stars--2 :nth-child(-n + 2),
+.stars--3 :nth-child(-n + 3),
+.stars--4 :nth-child(-n + 4),
+.stars--5 :nth-child(n) {
+  background-image: url(images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
```
background-position: center;
background-repeat: no-repeat;
```

These styles are not needed here as you are already setting them in the `stars__star`